Gems/EMotionFX/Code/EMotionFX/CommandSystem/Source/NodeGroupCommands.h

```diff
@@ -9,31 +9,83 @@
 #pragma once
 
 // include the required headers
+#include <AzCore/std/smart_ptr/unique_ptr.h>
 #include "CommandSystemConfig.h"
 #include <MCore/Source/Command.h>
 #include <MCore/Source/CommandGroup.h>
 #include <EMotionFX/Source/EMotionFXConfig.h>
+#include <EMotionFX/Source/NodeGroup.h>
+#include <EMotionFX/CommandSystem/Source/ParameterMixins.h>
 
 EMFX_FORWARD_DECLARE(Actor);
 EMFX_FORWARD_DECLARE(NodeGroup);
 
 namespace CommandSystem
 {
     // adjust a node group
-    MCORE_DEFINECOMMAND_START(CommandAdjustNodeGroup, "Adjust node group", true)
-    bool                    mOldDirtyFlag;
-    EMotionFX::NodeGroup*   mOldNodeGroup;
-    MCORE_DEFINECOMMAND_END
+    class CommandAdjustNodeGroup
+        : public MCore::Command
+        , public EMotionFX::ParameterMixinActorId
+    {
+    public:
+        AZ_CLASS_ALLOCATOR_DECL
+
+        enum class NodeAction
+        {
+            None,
```

Comment:
What's the use case for nodeAction::None vs use null option?

Reply:
Good call. I removed the `None` option.